test/inductor/test_pad_mm.py

```diff
@@ -11,14 +11,17 @@
     should_pad_common,
 )
 from torch._inductor.test_case import run_tests, TestCase
-from torch._inductor.utils import fresh_inductor_cache, run_and_get_code
+from torch._inductor.utils import fresh_inductor_cache, is_big_gpu, run_and_get_code
 from torch.testing import FileCheck
 from torch.testing._internal.inductor_utils import HAS_CUDA
 
 
 class PadMMTest(TestCase):
     @inductor_config.patch(max_autotune=True, max_autotune_gemm_backends="TRITON")
```

Comment:
Can you do this in the testSetup instead of in each test ? or just do it as a decorator on the class itself?

Reply:
@eellison neither option works directly. Decorators are executed on GPU-less host in RE so `is_big_gpu` just fails (I think). `testSetup` would affect all tests, but some tests here seem to not require `max_autotune=True` / TRITON backend to run. I could split the tests into two classes, but not sure what the behavior of `run_tests` would be: will it run tests from all classes in the file?

Reply:
On a second thought, I think, it's fine to check the `is_big_gpu` in `setUp`. Let me update the PR.